PandaGrom/hometask-5/4.3.sql

```diff
@@ -0,0 +1 @@
+SELECT CONCAT(last_name, '-', first_name) FROM actor WHERE first_name like '%W' OR last_name like 'H%'
```

Comment:
are you sure that here you select names starting with W and surnames ending with H?